chart/templates/tf_controller_role_binding.yaml

```diff
@@ -1,12 +1,27 @@
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding
 metadata:
-  name: terraform-controller-rolebinding
+  name: tf-controller-rolebinding
+  namespace: {{ .Release.Namespace }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
+  kind: Role
   name: tf-controller-role
 subjects:
 - kind: ServiceAccount
-  name: default
-  namespace: {{.Release.Namespace}}
+  name: tf-controller-service-account
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: tf-controller-read-provider-creds-rolebinding
+  namespace: {{ .Values.velaNamespace }}
```

Comment:
Should we use `.Release.Namespace` instead?

Reply:
The `hack/prepare-x-credentials.sh` scripts create the credentials secret in the `vela-system` namespace, so the idea of this role-binding is to allow the controller to access them when the controller is deployed in a different namespace.

If we changed this to `.Release.Namespace` instead, the controller may not be able to read the secrets